### PR TITLE
SQL breaking changes to fix Swagger spec validation errors/warnings

### PR DESCRIPTION
**Pull request by jaredmoo**
AutoRest enabled several new validation rules which exposed some naming issues in SQL swagger specs. I also noticed that several classes are no longer supported.
- Removed unsupported types (e.g. Schema, Table, Column)
- Revised import export  operation and class names
- Removed unnecessary import export operation results APIs

Corresponding Swagger change: https://github.com/Azure/azure-rest-api-specs/pull/1005

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [ ] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [x] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [x] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as a link to the swagger spec, used to generate the code.
- [x] The `project.json` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by cormacpayne** [comment]
This PR should be merged at the same time as https://github.com/Azure/azure-sdk-for-net/pull/2914 and https://github.com/Azure/azure-sdk-for-net/pull/2925

**Comment by jaredmoo** [comment]
#2914 and #2925 are now combined into this

**Comment by jaredmoo** [comment]
Swagger changes are all merged now.

**Comment by jaredmoo** [comment]
btw project.json isn't updated because 1.2.0-preview hasn't been published yet.

**Comment by cormacpayne** [comment]
@jaredmoo Hey Jared, two quick questions:

1) Just to be sure, does this spec include _all_ of the changes you originally wanted to make in this PR and the two that you closed?

2) There are a lot of unnecessary commits, would you mind [rebasing](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md#rebasing) to get the number of commits down to 1 or 2?

**Comment by jaredmoo** [comment]
Yes, since all swagger changes are now merged, all generated files in this change are based on rest-api-specs master branch.

I have squashed into a single commit and described all the breaking changes.

**Comment by jaredmoo** [comment]
@cormacpayne can I get a review?

File: global.json [hunk 1]
```diff
@@ -27,5 +27,8 @@
     "src/ResourceManagement/Insights/Microsoft.Azure.Insights",
     "src/ResourceManagement/AnalysisServices/Microsoft.Azure.Management.Analysis",
     "src/ResourceManagement/Monitor/Microsoft.Azure.Monitor"
-  ]
+  ],
+  "sdk": {
```
Comment (cormacpayne) [inline global.json]: @jaredmoo why was this added?

File: src/ClientRuntime/global.json [hunk 2]
```diff
@@ -1,5 +1,8 @@
 {
   "projects": [ "Microsoft.Rest.ClientRuntime", "Microsoft.Rest.ClientRuntime.Azure.Authentication", "Microsoft.Rest.ClientRuntime.Etw", 
                 "Microsoft.Rest.ClientRuntime.Tests", "Microsoft.Rest.ClientRuntime.Azure", "Microsoft.Rest.ClientRuntime.Azure.Tests",
-                "Microsoft.Rest.ClientRuntime.Log4Net", "Microsoft.Rest.ClientRuntime.Tracing.Tests" ]
+                "Microsoft.Rest.ClientRuntime.Log4Net", "Microsoft.Rest.ClientRuntime.Tracing.Tests" ],
+  "sdk": {
+    "version": "1.0.0-preview2-003121"
```
Comment (cormacpayne) [inline src/ClientRuntime/global.json]: @jaredmoo same comment

File: src/ResourceManagement/SqlManagement/global.json [hunk 3]
```diff
@@ -1,3 +1,6 @@
 {
-  "projects": [ "../../TestFramework", "Microsoft.Azure.Management.Sql", "Sql.Tests" ]
+  "projects": [ "../../TestFramework", "Microsoft.Azure.Management.Sql", "Sql.Tests" ],
+  "sdk": {
+    "version": "1.0.0-preview2-003121"
```
Comment (cormacpayne) [inline src/ResourceManagement/SqlManagement/global.json]: @jaredmoo same comment

File: src/TestFramework/global.json [hunk 4]
```diff
@@ -1,3 +1,6 @@
 {
-  "projects": [ "Microsoft.Rest.ClientRuntime.Azure.TestFramework", "Microsoft.Azure.Test.HttpRecorder" ]
+  "projects": [ "Microsoft.Rest.ClientRuntime.Azure.TestFramework", "Microsoft.Azure.Test.HttpRecorder" ],
+  "sdk": {
+    "version": "1.0.0-preview2-003121"
```
Comment (cormacpayne) [inline src/TestFramework/global.json]: @jaredmoo same comment

File: src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs [hunk 5]
```diff
@@ -127,24 +127,6 @@ public void TestImportExport(bool preexistingDatabase, string testPrefix, string
                         MaxSizeBytes = (2 * 1024L * 1024L * 1024L).ToString(),
                     });
                 }
-
-                // List Server import/export operations
```
Comment (cormacpayne) [inline src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs]: @jaredmoo is this removed block being tested somewhere else? Making sure there isn't any test regression with this being removed

File: src/ResourceManagement/SqlManagement/Sql.Tests/BlobAuditingTest.cs [hunk 6]
```diff
@@ -0,0 +1,183 @@
+﻿using Microsoft.Azure.Management.Resources;
```
Comment (cormacpayne) [inline src/ResourceManagement/SqlManagement/Sql.Tests/BlobAuditingTest.cs]: @jaredmoo this file is missing the license header

File: src/ResourceManagement/SqlManagement/Sql.Tests/ThreatDetectionTest.cs [hunk 7]
```diff
@@ -0,0 +1,166 @@
+﻿using Microsoft.Azure.Management.Resources;
```
Comment (cormacpayne) [inline src/ResourceManagement/SqlManagement/Sql.Tests/ThreatDetectionTest.cs]: @jaredmoo this file is missing license header

**Review by cormacpayne — CHANGES_REQUESTED** [review]
@jaredmoo some comments that need to be resolved

Also, what is the `UpgradeLog.htm` file? I was unable to view it on GitHub or CodeFlow

File: global.json (hunk 1 above)
Comment (jaredmoo) [inline global.json]: I have VS2017 installed on my machine, without this the build is totally broken. See https://github.com/Azure/azure-sdk-for-net/issues/2922 .

File: src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs (hunk 5 above)
Comment (jaredmoo) [inline src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs]: This API is removed so no test is needed.

Comment (jaredmoo) [inline src/ResourceManagement/SqlManagement/Sql.Tests/ImportExportScenarioTests.cs]: Fixed in 83501016bb0744278a7d4a388985c2c37fc80258

File: src/ResourceManagement/SqlManagement/Sql.Tests/ThreatDetectionTest.cs (hunk 7 above)
Comment (jaredmoo) [inline src/ResourceManagement/SqlManagement/Sql.Tests/ThreatDetectionTest.cs]: Fixed in 83501016bb0744278a7d4a388985c2c37fc80258

File: src/ClientRuntime/global.json (hunk 2 above)
Comment (jaredmoo) [inline src/ClientRuntime/global.json]: #2922

File: src/ResourceManagement/SqlManagement/global.json (hunk 3 above)
Comment (jaredmoo) [inline src/ResourceManagement/SqlManagement/global.json]: #2922

File: src/TestFramework/global.json (hunk 4 above)
Comment (jaredmoo) [inline src/TestFramework/global.json]: #2922

File: src/ResourceManagement/SqlManagement/Sql.Tests/BlobAuditingTest.cs (hunk 6 above)
Comment (jaredmoo) [inline src/ResourceManagement/SqlManagement/Sql.Tests/BlobAuditingTest.cs]: Fixed in 8350101

**Comment by jaredmoo** [comment]
`UpgradeLog.htm` is visual studio junk. Delete it.

**Comment by jaredmoo** [comment]
@cormacpayne - I have addressed feedback. Not changing project.json since then it doesn't build for me. I think project.json goes away when #2815 is done.

**Comment by cormacpayne** [comment]
@jaredmoo it looks like there is a merge conflict from a pull request that was merged earlier today

**Comment by jaredmoo** [comment]
Ok, I reverted my change to this file in order to resolve the conflict.

**Comment by jaredmoo** [comment]
@cormacpayne ready to merge again :)

**Review by cormacpayne — APPROVED** [review]

**Pull request merged**